### PR TITLE
Add links to Summarizer and Gallery for Python

### PR DESCRIPTION
**Pull request by achandmsft**
@bobtabor-msft please review and wordsmith if needed.

### CODE REVIEW

File: articles/intro/includes/azure-ai-for-developers-python.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Azure AI reference templates provide you with well-maintained, easy to deploy re
 |[Build a chat app with Azure OpenAI in Python](https://github.com/Azure-Samples/chatgpt-quickstart/blob/main/README.md)|A simple Python Quart app that streams responses from ChatGPT to an HTML/JS frontend using JSON Lines over a ReadableStream.|
 |[Build a LangChain with Azure OpenAI in Python](https://github.com/Azure-Samples/function-python-ai-langchain)|An Azure Functions sample that shows how to take a human prompt as HTTP Get or Post input, calculates the completions using chains of human input and templates. This is a starting point that can be used for more sophisticated chains.|
 |[Build a ChatGPT Plugin with Azure Container Apps in Python](https://github.com/Azure-Samples/openai-plugin-fastapi/blob/main/README.md)|A sample for creating ChatGPT Plugin using GitHub Codespaces, VS Code, and Azure. The sample includes templates to deploy the plugin to Azure Container Apps using the Azure Developer CLI.|
+|[Summarize Text using Azure AI Language with Azure Functions](https://github.com/Azure-Samples/function-python-ai-textsummarize)|Take text documents as input, summarize using Azure AI Language, and then output to another text document using Azure Functions|
```
Comment (bobtabor-msft) [inline articles/intro/includes/azure-ai-for-developers-python.md:18]: Tiny nitpick ... probably should add periods at the end of each of these sentences in the second column. Not critical.

**Review by bobtabor-msft — APPROVED** [review]

File: articles/intro/includes/azure-ai-for-developers-python.md (hunk 1 above)
Comment (Court72) [inline articles/intro/includes/azure-ai-for-developers-python.md:18]:
```suggestion
|[Summarize Text using Azure AI Language with Azure Functions](https://github.com/Azure-Samples/function-python-ai-textsummarize)|Take text documents as input, summarize using Azure AI Language, and then output to another text document using Azure Functions.|
```

File: articles/intro/includes/azure-ai-for-developers-python.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ Azure AI reference templates provide you with well-maintained, easy to deploy re
...
+|[Azure AI Python Template Gallery](https://aka.ms/azai/py/gallery)|For the full list of Azure AI templates, visit our gallery. All app templates in our gallery can be spun up and deployed using a single command: _azd up_|
```
Comment (Court72) [inline articles/intro/includes/azure-ai-for-developers-python.md:19]:
```suggestion
|[Azure AI Python Template Gallery](https://aka.ms/azai/py/gallery)|For the full list of Azure AI templates, visit our gallery. All app templates in our gallery can be spun up and deployed using a single command: _azd up_.|
```

**Comment by Court72** [comment]
@bobtabor-msft

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by bobtabor-msft** [comment]
We're re-creating these changes and adding some additional one on the private repo. Closing.

**Pull request closed without merging**